eng/scripts/docs/Docs-ToC.ps1

```diff
@@ -21,3 +21,178 @@ function Get-NamepspacesFromDll($dllPath) {
     }
     return $namespaces | Sort-Object -Unique
 }
+
+function DownloadNugetPackage($package, $version, $destination) {
+    # $PackageSourceOverride is a global variable provided in
+    # Update-DocsMsToc.ps1. Its value can set a "customSource" property.
+    # If it is empty then the property is not overridden.
+    $customPackageSource = Get-Variable -Name 'PackageSourceOverride' -ValueOnly -ErrorAction 'Ignore'
+    # Download package from nuget, if no package found in nuget, then search devops public feeds if set.
+    Write-Host "Downloading $package with version $version..."
+    if ($customPackageSource) {
```

Comment:
Should we handle `$LASTEXITCODE` for these nuget calls?

Reply:
Don't want to check the error code in each individual nuget command, so I have error handler when we call the Download function.